components/widgets/inputs/users_emails_input.jsx

```diff
@@ -278,6 +278,7 @@ class UsersEmailsInput extends React.Component {
                 onFocus={this.onFocus}
                 tabSelectsValue={true}
                 value={values}
+                aria-label={this.props.placeholder}
```

Comment:
This should be generic, and I don't think the `aria-label` should be bound to the `placeholder` property, I would request (as mandatory) a new `ariaLabel` property in the `UsersEmailsInput` (and in the `ChannelsInput` component), and then, from the outside I would use the `placeholder` if it makes sense.